core/src/tile/tileBuilder.cpp

```diff
@@ -141,7 +141,7 @@ std::shared_ptr<Tile> TileBuilder::build(TileID _tileID, const TileData& _tileDa
         tile->setMesh(builder.second->style(), builder.second->build());
     }
 
-    tile->setSelectionFeatures(m_selectionFeatures);
+    tile->setSelectionFeatures(std::move((m_selectionFeatures)));
 
```

Comment:
"move does not move" :) 

The copy is intended so that m_selectionFeatures does not get constantly reallocated but is used as a buffer. By copying the selectionFeatures hold by Tile also gets shrinked to the appropriate size.